File: 프로그래머스/1/134240. 푸드 파이트 대회/푸드 파이트 대회.java
class Solution {
    public String solution(int[] food) {
        String ret = "0";
        for(int i = food.length - 1 ; i > 0; i--){
            food[i] -= food[i] % 2;
            if(food[i] == 0) continue;
            for(int j = 0; j < food[i]; j += 2){
                ret = "" + i + ret;
                ret = ret + "" + i;
            }
        }
        return ret;
    }
}

File: 프로그래머스/1/134240. 푸드 파이트 대회/푸드 파이트 대회.py
def solution(food):
    ret = '0'
    for i in range(len(food) - 1, 0, -1):
        for _ in range(food[i]//2):
            ret = str(i) + ret
            ret += str(i)
    return ret
    
"""
중앙 물먹으면 승리
칼로리 낮은 것부터 먹음,
대칭,

"""